sysl2/sysl/seqs/visitor.go

```diff
@@ -207,17 +232,21 @@ func (v *SequenceDiagramVisitor) Visit(e Element) error {
 			log.Errorln(err)
 		}
 	}()
-
+	var err error
 	switch t := e.(type) {
 	case *EndpointCollectionElement:
-		return v.visitEndpointCollection(t)
+		err = v.visitEndpointCollection(t)
+		for bbKey, bbVal := range t.blackboxes {
+			if bbVal.ValueType == BBEndpointCollection && bbVal.VisitCount == 0 {
+				log.Warnf("blackbox '%s' not hit in app %s\n", bbKey, v.currentApp)
+			}
+		}
 	case *EndpointElement:
-		return v.visitEndpoint(t)
+		err = v.visitEndpoint(t)
```

Comment:
```go
return v.visitEndpoint(t)
```

Reply:
I could return from v.visitEndpointCollection/v.visitEndpoint, but golint then starts complaining about a missing return at the end of statement: missing return (typecheck)